libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt

```diff
@@ -730,17 +730,20 @@ class MapboxNavigationViewportDataSource(
             } else {
                 val mapSize = mapboxMap.getSize()
                 val screenBox = getScreenBoxForFraming(mapSize, followingPadding)
-
-                val fallbackCameraOptions = mapboxMap.getCameraOptions()
-                    .toBuilder()
+                val fallbackCameraOptions = mapboxMap.cameraForCoordinates(
+                    listOf(mapboxMap.cameraState.center),
+                    mapboxMap.cameraState.padding,
+                    mapboxMap.cameraState.bearing,
+                    mapboxMap.cameraState.pitch
+                ).toBuilder()
                     .center(
                         pointsForFollowing.firstOrNull()
-                            ?: mapboxMap.getCameraOptions().center!!
+                            ?: mapboxMap.cameraState.center
                     )
                     .padding(getMapAnchoredPaddingFromUserPadding(mapSize, followingPadding))
                     .bearing(followingBearingProperty.get())
                     .pitch(followingPitchProperty.get())
-                    .zoom(mapboxMap.getCameraOptions().zoom!!)
+                    .zoom(mapboxMap.cameraState.zoom)
```

Comment:
Changes here are incorrect, we should use something like `mapboxMap.cameraState.toBuilder`.

Reply:
There is no CameraState.toBuilder().